releasenotes.txt

```diff
@@ -25,10 +25,19 @@ Other:
 - Multiple "--project" options are now prohibited. These would have overlapped each other and lead to unexpected behavior. Please use separate runs with a single "--project" option instead.
 - "--project" can also no longer be used in conjunction with additional source files.
 - If a addon cannot be found it will bail out immediately instead of continously writing errors and failing the analysis at the end.
+- clang-tidy was not invoked when the process-based executor (only available on Linux / default) was used.
 - Added CMake option "BUILD_MANPAGE" which adds the "man" target which will build the manpage. This requires xsltproc and the docbook XSLs to be installed.
+- Added some library configurations to the official Windows installer.
+- Removed some unused Qt DLLs from the official Windows installer.
 - Improved inline suppressions:
   - You can suppress warnings in a block of code using "-begin" and "-end".
   - You can suppress warnings in current file using "-file".
   - You can suppress all warnings where macro is used using "-macro"
+- Cppcheck will now error out when an option, which requires code which is not compiled in, is being specified. In the past it might have silently ignored it or reported it as unknown.
+- Several macros defined by "limits.h"/"climits" will now be generated from the used platform condiguration and passed to the analysis.
 - fixed CMake build with UBSAN and GCC
+- Files will now be processed in the order as specified via the command-line.
 - Added command-line options "--fsigned-char" and "--funsigned-char" to control the signess of the "char" type. This overrides previously specified "--platform" options and is overrides by following ones.
+- Markup files will now be processed after the regular source files when using multiple threads/processes (some issues remain - see Trac #12167 for details).
+- Added file name to ValueFlow "--debug" output.
```

Comment:
This is something only cppcheck developers might care about right? I'm not sure if I would write that info here.